src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfCreator.java

```diff
@@ -34,9 +37,30 @@ public PdfCreator(DuplexPreparator duplexPreparator, IHtmlToPdfConverter convert
         return PdfMerger.mergeDocuments(docs);
     }
 
+    public byte[] create(LetterWithPdfsRequest letter) {
```

Comment:
If all we need is the documents from the letter then we could take the documents as a parameter

Reply:
We could change that method too